src/util/object_factory_parameters.cpp

```diff
@@ -74,6 +74,11 @@ void parse_object_factory_options(const cmdlinet &cmdline, optionst &options)
       "max-nondet-string-length",
       cmdline.get_value("max-nondet-string-length"));
   }
+  else if(!cmdline.isset("no-refine-strings"))
```

Comment:
Suggest add blank lines to make clear we've hidden an `if/else` amongst this sequence of `if`s

Reply:
I get your point, although I would really like to merge this PR and the test-gen bump today (to avoid even more accumulation for the bump over the weekend) so I will not update this.

Reply:
```suggestion
  else
```
I don't think the `if` part is needed.

Reply:
That would cause the max length to be bounded if --no-refine-strings is given which I think is not desired. And it would be bounded by a reasonably small value. If anyone would like to increase it, while keeping --no-refine-strings, they would get an error as one cannot define both options at the same time.

Reply:
@smowton I had to restart the CI anyway so I applied your suggestion